pagerduty/resource_pagerduty_extension.go

```diff
@@ -98,6 +111,10 @@ func resourcePagerDutyExtensionRead(d *schema.ResourceData, meta interface{}) er
 	}
 	d.Set("extension_schema", extension.ExtensionSchema)
 
+	if err := d.Set("config", flattenExtensionConfig(extension.Config)); err != nil {
```

Comment:
The schema expects a `string` here but is actually a `[]byte` so this won't work. 
Think you must wrap a `string()` here or have the flatten func return a string.

Reply:
Well, it does work as is as the acceptance tests pass.

Also, I'm using it with success on our own stack.

Reply:
I'm sure it works correctly but the problem here is that we're failing to set `config` as returned by the API (it's never updated).

Note: I'm seeing `error setting extension config` when I'm running the acctests.

Reply:
That makes sense to me now, thanks!